proto/store/setting.proto

```diff
@@ -74,8 +76,45 @@ message SMTPMailDeliverySetting {
   }
   Authentication authentication = 7;
   string username = 8;
-  string password = 9;
+  optional string password = 9;
 
   // The sender email address.
   string from = 10;
+
+  // The recipient email address, used with validate_only to send test email.
+  optional string to = 11;
```

Comment:
`to` is only use in test email, we would never store it.

Reply:
We're not https://github.com/bytebase/bytebase/pull/6070/files#diff-2ed74e966bf7e4da7fcf16edd5f25cd8526a65d2e97c89f5adf9907b570da5ddR200